### ISSUE TITLE
Select - WCAG 2.5.3 Label in Name

### ISSUE THREAD

**Issue by britt6612** [grommet/grommet#7646]
### Is this something you or your team is interested in contributing a pull request towards?

None

### Expected Behavior

Currently the Select component is failing the wcag rule 2.5.3 The visual label for controls is a trigger for speech activation.

- [ ] One the issue is resolved go back and update the rule in DS site repo

### Actual Behavior

its not passing

**Comment by jcfilben** [comment]
Need to update the following rules on DS once this is resolved:

- Select 2.5.3
- SelectMultiple 2.5.3
- DataSort 2.5.3 (looks like this is incorrectly marked as passing)
- DataFilter 2.5.3
- DataFilters 2.5.3

**Comment by jcfilben** [comment]
Possibly related to https://github.com/grommet/grommet/issues/5167
